symfony/plugins/orangehrmAttendancePlugin/Dao/AttendanceDao.php

```diff
@@ -756,4 +758,65 @@ public function getTotalWorkingTime(AttendanceRecordSearchFilterParams $attendan
 
         return $q->getQuery()->getOneOrNullResult();
     }
+
+    /**
+     * @param EmployeeAttendanceSummarySearchFilterParams $employeeAttendanceSummarySearchFilterParams
+     * @return array
+     */
+    public function getEmployeeAttendanceSummaryList(EmployeeAttendanceSummarySearchFilterParams $employeeAttendanceSummarySearchFilterParams): array
+    {
+        return $this->getEmployeeAttendanceSummaryPaginator($employeeAttendanceSummarySearchFilterParams)->getQuery()->execute();
+    }
+
+    /**
+     * @param EmployeeAttendanceSummarySearchFilterParams $employeeAttendanceSummarySearchFilterParams
+     * @return int
+     */
+    public function getEmployeeAttendanceSummaryListCount(
+        EmployeeAttendanceSummarySearchFilterParams $employeeAttendanceSummarySearchFilterParams
+    ): int {
+        $paginator = $this->getEmployeeAttendanceSummaryPaginator($employeeAttendanceSummarySearchFilterParams);
+        return $paginator->count();
+    }
+
+    /**
+     * @param EmployeeAttendanceSummarySearchFilterParams $employeeAttendanceSummarySearchFilterParams
+     * @return Paginator
+     */
+    private function getEmployeeAttendanceSummaryPaginator(EmployeeAttendanceSummarySearchFilterParams $employeeAttendanceSummarySearchFilterParams): Paginator
+    {
+        $q = $this->getEmployeeAttendanceSummaryQueryBuilderWrapper($employeeAttendanceSummarySearchFilterParams)->getQueryBuilder();
+        $q->select(
+            'CONCAT(employee.firstName, \' \', employee.lastName) AS fullName',
```

Comment:
better to split these params, cz this is not report API

Reply:
done